tasks/main.yml

```diff
@@ -20,5 +20,20 @@
   tags:
     - haproxy
 
+- name: Make sure service dropin directory is available
+  file:
+    path: /etc/systemd/system/marathon.service.d/
+    state: directory
+  when: ansible_service_mgr == 'systemd'
```

Comment:
If I add this check, the next task fails (which creates the ssl.conf). This is fixable, but now the config file is removed when `mesos_ssl_enabled` is set to false, resulting in a setup that is fully working again without HTTPS